nuxeo-customer-project-sample-package/src/main/assemble/assembly.xml

```diff
@@ -1,31 +1,31 @@
 <project name="custom-package" default="build" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
-  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build"/>
-  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact"/>
-  <taskdef resource="net/sf/antcontrib/antlib.xml"/>
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build" />
+  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact" />
+  <taskdef resource="net/sf/antcontrib/antlib.xml" />
 
   <target name="build" description="Build Nuxeo Package">
-    <property name="outdir" value="${maven.project.build.directory}"/>
+    <property name="outdir" value="${maven.project.build.directory}" />
     <!-- Include your artifacts in the graph adding your groupId to groupPrefixes -->
-    <artifact:nuxeo-expand includeTestScope="true" groupPrefixes="org.nuxeo,com.customer.sample"/>
-    <artifact:print output="${outdir}/custom-package-artifacts.properties" mode="sdk"/>
-    <artifact:print output="${outdir}/custom-package-test-artifacts.properties" mode="sdk" scopes="test"/>
+    <artifact:nuxeo-expand includeTestScope="true" groupPrefixes="org.nuxeo,com.customer.sample" />
+    <artifact:print output="${outdir}/custom-package-artifacts.properties" mode="sdk" />
+    <artifact:print output="${outdir}/custom-package-test-artifacts.properties" mode="sdk" scopes="test" />
     <!-- Prints the graph for debugging purpose -->
-    <artifact:print output="${outdir}/dependency-tree.log"/>
-    <tstamp/>
+    <artifact:print output="${outdir}/dependency-tree.log" />
+    <tstamp />
 
-    <delete failonerror="false" dir="${outdir}/nxr"/>
-    <mkdir dir="${outdir}/nxr"/>
-    <delete failonerror="false" dir="${outdir}/marketplace"/>
-    <mkdir dir="${outdir}/marketplace"/>
+    <delete failonerror="false" dir="${outdir}/nxr" />
+    <mkdir dir="${outdir}/nxr" />
+    <delete failonerror="false" dir="${outdir}/marketplace" />
+    <mkdir dir="${outdir}/marketplace" />
     <copy todir="${outdir}/marketplace">
-      <fileset dir="src/main/resources"/>
+      <fileset dir="src/main/resources" />
       <filterset>
-        <filter token="VERSION" value="${maven.project.version}"/>
-        <filter token="DISTRIB_VERSION" value="${nuxeo.distribution.version}"/>
+        <filter token="VERSION" value="${maven.project.version}" />
+        <filter token="DISTRIB_VERSION" value="${nuxeo.platform.version}" />
```

Comment:
```
        <filter token="NUXEO_TARGET_VERSION" value="${nuxeo.target.version}" />
```
with
```
<nuxeo.target.version>11.*</nuxeo.target.version>
```
would allow to install the package on 11.* and not only `11.2.13` which is the exact version pointed by nuxeo-parent:11.2.

See https://github.com/nuxeo/nuxeo-explorer/blob/master/pom.xml#L17 and https://github.com/nuxeo/nuxeo-explorer/blob/master/packages/nuxeo-platform-explorer-package/src/main/resources/package.xml#L11.